contracts/staking/StakingStorage.sol

```diff
@@ -87,3 +87,9 @@ contract StakingV2Storage is StakingV1Storage {
     // Destination of accrued rewards : beneficiary => rewards destination
     mapping(address => address) public rewardsDestination;
 }
+
+contract StakingV3Storage is StakingV2Storage {
+    // Additional rebate parameters for exponential rebates
+    uint32 public lambdaNumerator;
```

Comment:
I think these two fit right after `alphaDenominator` without breaking the slot order.

Reply:
I think they don't fit, slot 13 (see below) has the first 28 bytes already taken, so we only have the last 4 bytes to play with which is enough to fit 1 uint32 but not both. Unless we turn them into uint16, which could work but would be weird on it's own.

Not sure which one is "worse":

- Option A: add `lambdaNumerator` and `lambdaDenominator` on new storage after all `StakingV2Storage`, keep them as uint32, consistent with their alpha counterparts
- Option B: shrink `lambdaNumerator` and `lambdaDenominator` to uint16 and put them into slot 13.



```
┌────────────────────────────┬─────────────────────────────────────┬──────────────┬────────┬───────────────────────────────────────────────────────────────────────────────┐
│ contract                   │ state_variable                      │ storage_slot │ offset │ type                                                                          │
├────────────────────────────┼─────────────────────────────────────┼──────────────┼────────┼───────────────────────────────────────────────────────────────────────────────┤
│ StakingV3Storage           │ controller                          │      0       │   0    │ t_contract(IController)16529                                                  │
│ StakingV3Storage           │ _addressCache                       │      1       │   0    │ t_mapping(t_bytes32,t_address)                                                │
│ StakingV3Storage           │ __gap                               │      2       │   0    │ t_array(t_uint256)10_storage                                                  │
│ StakingV3Storage           │ minimumIndexerStake                 │      12      │   0    │ t_uint256                                                                     │
│ StakingV3Storage           │ thawingPeriod                       │      13      │   0    │ t_uint32                                                                      │
│ StakingV3Storage           │ curationPercentage                  │      13      │   4    │ t_uint32                                                                      │
│ StakingV3Storage           │ protocolPercentage                  │      13      │   8    │ t_uint32                                                                      │
│ StakingV3Storage           │ __DEPRECATED_channelDisputeEpochs   │      13      │   12   │ t_uint32                                                                      │
│ StakingV3Storage           │ maxAllocationEpochs                 │      13      │   16   │ t_uint32                                                                      │
│ StakingV3Storage           │ alphaNumerator                      │      13      │   20   │ t_uint32                                                                      │
│ StakingV3Storage           │ alphaDenominator                    │      13      │   24   │ t_uint32                                                                      │
│ StakingV3Storage           │ stakes                              │      14      │   0    │ t_mapping(t_address,t_struct(Indexer)25153_storage)                           │
│ StakingV3Storage           │ allocations                         │      15      │   0    │ t_mapping(t_address,t_struct(Allocation)20139_storage)                        │
│ StakingV3Storage           │ subgraphAllocations                 │      16      │   0    │ t_mapping(t_bytes32,t_uint256)                                                │
│ StakingV3Storage           │ __DEPRECATED_rebates                │      17      │   0    │ t_mapping(t_uint256,t_uint256)                                                │
│ StakingV3Storage           │ slashers                            │      18      │   0    │ t_mapping(t_address,t_bool)                                                   │
│ StakingV3Storage           │ delegationRatio                     │      19      │   0    │ t_uint32                                                                      │
│ StakingV3Storage           │ delegationParametersCooldown        │      19      │   4    │ t_uint32                                                                      │
│ StakingV3Storage           │ delegationUnbondingPeriod           │      19      │   8    │ t_uint32                                                                      │
│ StakingV3Storage           │ delegationTaxPercentage             │      19      │   12   │ t_uint32                                                                      │
│ StakingV3Storage           │ delegationPools                     │      20      │   0    │ t_mapping(t_address,t_struct(DelegationPool)20161_storage)                    │
│ StakingV3Storage           │ operatorAuth                        │      21      │   0    │ t_mapping(t_address,t_mapping(t_address,t_bool))                              │
│ StakingV3Storage           │ assetHolders                        │      22      │   0    │ t_mapping(t_address,t_bool)                                                   │
│ StakingV3Storage           │ rewardsDestination                  │      23      │   0    │ t_mapping(t_address,t_address)                                                │
│ StakingV3Storage           │ lambdaNumerator                     │      24      │   0    │ t_uint32                                                                      │
│ StakingV3Storage           │ lambdaDenominator                   │      24      │   4    │ t_uint32                                                                      │
```